src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -174,38 +176,52 @@ const AugmentedRealityView = ({ sceneNavigator }) => {
 
   // TODO: these data can be updated according to the data coming from the server when the
   //       real 3D models arrive
-  const position = [0, -1, -5];
+  const position = [0, 0, 0];
   const rotation = [0, 0, 0];
-  const scale = [0.02, 0.02, 0.02];
+  const scale = [0.002, 0.002, 0.002];
+
+  ViroARTrackingTargets.createTargets({
+    targetImage: {
+      physicalWidth: 0.2, // real world width in meters
```

Comment:
is this also something that could change depending on the ar model or given trigger image thus it needs to be defined per server json?

Reply:
I guess we don't need to add information about it in the json.